centaur/src/main/resources/standardTestCases/globbing_behavior_with_requester_pays.test

```diff
@@ -0,0 +1,13 @@
+name: globbing_behavior_with_requester_pays
+testFormat: workflowsuccess
+backends: [Papiv2]
+
+files {
+  workflow: globbing_behavior_with_requester_pays/globbing_behavior_with_requester_pays.wdl
+}
+
+metadata {
+  workflowName: globbing_behavior_with_requester_pays
+  status: Succeeded
+  "outputs.globbing_behavior_with_requester_pays.globArrayContent": "globArray"
+}
```

Comment:
looks like you need some of that ["always final newline"](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file) IntelliJ magic